src/tasks-prisma.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from './prisma.service';
+import { prisma, Prisma, task, user } from '@prisma/client';
```

Comment:
These variables may not be currently in use.
```suggestion
import { task, user } from '@prisma/client';
```